File: app/src/main/java/fr/corenting/edcompanion/activities/AboutActivity.java
package fr.corenting.edcompanion.activities;

import android.content.pm.PackageInfo;
import android.content.pm.PackageManager;
import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.support.v7.app.AppCompatDelegate;
import android.support.v7.widget.Toolbar;
import android.text.Html;
import android.text.method.LinkMovementMethod;
import android.view.MenuItem;
import android.view.View;
import android.widget.TextView;

import fr.corenting.edcompanion.R;
import fr.corenting.edcompanion.utils.ThemeUtils;


public class AboutActivity extends AppCompatActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {

        // Set theme first before parent call
        AppCompatDelegate.setDefaultNightMode(ThemeUtils.getDarkThemeValue(this));

        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_about);

        //Action bar setup
        Toolbar mToolbar = (Toolbar) findViewById(R.id.toolbar);
        setSupportActionBar(mToolbar);
        getSupportActionBar().setDisplayHomeAsUpEnabled(true);
        ThemeUtils.setToolbarColor(this, mToolbar);

        TextView versionTextView = (TextView) findViewById(R.id.versionTextView);
        TextView libsContentTextView = (TextView) findViewById(R.id.librariesContentTextView);
        TextView contactTextView = (TextView) findViewById(R.id.contactTextView);

        PackageManager manager = getPackageManager();
        PackageInfo info;
        String version = " version ";
        try {
            info = manager.getPackageInfo(getPackageName(), 0);
            version += info.versionName;
        } catch (PackageManager.NameNotFoundException e) {
            versionTextView.setVisibility(View.GONE);
        }
        versionTextView.setText(version);
        setHtmlTextView(libsContentTextView, getString(R.string.about_libs));
        setHtmlTextView(contactTextView, getString(R.string.about_contact_me));
    }

    @Override
    public boolean onOptionsItemSelected(MenuItem item) {
        int id = item.getItemId();
        if (id == android.R.id.home) {
            finish();
        }
        return super.onOptionsItemSelected(item);
    }

    private static void setHtmlTextView(TextView textView, String text) {
        textView.setText(Html.fromHtml(text));
        textView.setMovementMethod(LinkMovementMethod.getInstance());
    }
}

File: app/src/main/java/fr/corenting/edcompanion/activities/AboutActivity.kt
package fr.corenting.edcompanion.activities

import android.content.pm.PackageInfo
import android.content.pm.PackageManager
import android.os.Bundle
import android.text.method.LinkMovementMethod
import android.view.MenuItem
import android.view.View
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity
import androidx.appcompat.app.AppCompatDelegate
import androidx.core.text.HtmlCompat
import fr.corenting.edcompanion.R
import fr.corenting.edcompanion.databinding.ActivityAboutBinding
import fr.corenting.edcompanion.utils.ThemeUtils

class AboutActivity : AppCompatActivity() {

    private lateinit var binding: ActivityAboutBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        AppCompatDelegate.setDefaultNightMode(ThemeUtils.getThemeToUse(this))
        super.onCreate(savedInstanceState)
        binding = ActivityAboutBinding.inflate(layoutInflater)
        val view = binding.root
        setContentView(view)

        //Action bar setup
        setSupportActionBar(binding.includeAppBar.toolbar)
        supportActionBar?.setDisplayHomeAsUpEnabled(true)

        var version: String? = " version "
        try {
            val info: PackageInfo = packageManager.getPackageInfo(packageName, 0)
            version += info.versionName
        } catch (e: PackageManager.NameNotFoundException) {
            binding.versionTextView.visibility = View.GONE
        }
        binding.versionTextView.text = version
        setHtmlTextView(binding.librariesContentTextView, getString(R.string.about_libs))
        setHtmlTextView(binding.iconsContentTextView, getString(R.string.about_icons))
        setHtmlTextView(binding.contactTextView, getString(R.string.about_contact_me))
    }

    override fun onOptionsItemSelected(item: MenuItem): Boolean {
        when (item.itemId) {
            android.R.id.home -> {
                finish()
            }
        }
        return super.onOptionsItemSelected(item)
    }

    companion object {
        private fun setHtmlTextView(textView: TextView, text: String) {
            textView.text = HtmlCompat.fromHtml(text, HtmlCompat.FROM_HTML_MODE_LEGACY)
            textView.movementMethod = LinkMovementMethod.getInstance()
        }
    }
}